CaloConditions/src/CalCalibMaker.cc

```diff
@@ -0,0 +1,29 @@
+#include "Offline/CaloConditions/inc/CalCalibMaker.hh"
+#include <vector>
+
+/*
+Written by S. Middleton
+*/
+namespace mu2e {
+  typedef std::shared_ptr<CalCalib> ptr_t;
+
+  ptr_t CalCalibMaker::fromFcl() {
+  CalCalibPar nominal(_config.ADC2MeV(), _config.timeoffset(), _config.algName());
+
+  size_t nChan = 1; //FIXME
+
+  CalCalib::CalibVec cvec(nChan, nominal);
+  auto ptr = std::make_shared<CalCalib>(cvec);
+  return ptr;
+
+  } // end fromFcl
+
+  ptr_t CalCalibMaker::fromDb(CalEnergyCalib::cptr_t) {
+    // initially fill from fcl to get all the constants
+
+    auto ptr = fromFcl();
```

Comment:
I expected this to contain code to fill CalCalib from the db table argument. It is fine if you want to write that code later.

Reply:
yes it will be part of the next iteration